File: implement-the-mutations-algorithm.py
def mutation(arr):
    target = arr[0].lower()
    test = arr[1].lower()

    for char in test :
        if char not in target:
            return False
    return True

#Variant..
def mutation_smart(arr):
    return all(char in arr[0].lower() for char in arr[1].lower())

print(mutation(["alien","L1NE"]))

File: implement-the-mutations-algorithm.js
function mutation(arr) {
    const target = arr[0].toLowerCase();
    const test = arr[1].toLowerCase();

    for (let i =0; i< test.length; i++) {
        if(target.indexOf(test[i]) === -1) {
            return false;
        }
    }
    return true;
}

//Variant..

function mutation(arr) {
    const target = arr[0].toLowerCase();
    const test = arr[1].toLowerCase();

    for (let i=0; i<test.length; i++) {
        if (!target.includes(test[i])) {
            return false;
        }
    }
    return true;
}